internal/auth/firebase.go

```diff
@@ -163,6 +163,23 @@ func (f *firebaseAuth) CreateUser(ctx context.Context, name, email, pass string,
 	return true, nil
 }
 
+// DisableUser prevents a user from logging in by disabling their account.
+func (f *firebaseAuth) DisableUser(ctx context.Context, email string) error {
```

Comment:
Why do we need to disable the user in firebase versus doing a local disable in the database?

Reply:
if there were an abusive user this stops all access, it's also flippable so you could disable and then enable without having to re-join all their realms

Reply:
We can set `DeletedAt` on the user in the database and achieve the same effect (without removing them from all the realms). I'm worried about a situation where the developer's firebase is attached to multiple systems. I dont think they'd expect disabling a user here to disable them in other systems.